pgjdbc/src/main/java/org/postgresql/PGProperty.java

```diff
@@ -173,6 +173,12 @@
   SSL_FACTORY_ARG("sslfactoryarg", null,
       "Argument forwarded to constructor of SSLSocketFactory class."),
 
+   /**
+    * The String argument to give to the constructor of the SSL Context Factory.
+    */
+   SSL_CONTEXT_PROTOCOL("sslContextProtocol", "TLS",
+          "Argument forwarded to constructor of SSLContext. Explicit SSL protocol to use when creating an SSLContext."),
```

Comment:
I can definitely add that to the comments as well to make that clearer I'll also add it to the documentation as requested by @davecramer